### PR TITLE
:version:() update TREX to solidity version 0.6.0

### PR DESCRIPTION
**Pull request by Joachim-Lebrun**

### CODE REVIEW

**Comment by Joachim-Lebrun** [comment]
there is a problem with the onchainID import, it says that the compiler version is still 0.5.10 which is weird as i changed the `@onchain-id/solidity` version to 0.4.0 in the `package.json`

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/TokenySolutions/T-REX#README",
   "devDependencies": {
     "@commitlint/cli": "^8.3.5",
-    "@onchain-id/solidity": "0.3.1",
+    "@onchain-id/solidity": "0.4.0",
```
Comment (Joachim-Lebrun) [inline package.json]: there is a problem with this dependency apparently ...

Comment (Nakasar) [reply package.json]: I feel so dumb >>'
The published package had the wrong version of the repo, I have released a new version 0.4.1 with the proper contracts, sorry!

**Comment by Joachim-Lebrun** [comment]
i still need to do the test to go back to 100% coverage, after that we should be able to merge on `master`

**Comment by Joachim-Lebrun** [comment]
no idea why, but there is a bug on github and this commit https://github.com/TokenySolutions/T-REX/commit/639c074d2938082152fa4a66a884d7bb6ade117b is not in the PR

**Comment by Myna65** [comment]
I'm closing and opening a one new since I'm not sure that the commit mentonned by @Joachim-Lebrun  will be included or not

**Comment by Myna65** [comment]
Replaced by #42

**Pull request closed without merging**